study-group/Toast-Implementation-Studies.md

```diff
@@ -10,8 +10,8 @@ The libraries studied are:
 - [Material-UI Snackbar](#material-ui-snackbar)
 - [Android `Toast`](#android-toast)
 - [react-toastify](#react-toastify)
-- [ngx-toastr](#ngx-toastr)
-- [Bootstrap](#bootstrap)
+- [ngx-toastr](#ngx-toast)
```

Comment:
Was this change intended?

Reply:
Probably not, I think that came from some of the merge shenanigans
```suggestion
- [ngx-toastr](#ngx-toastr)
```